site/en/reference/cli/concepts/srp.md

```diff
@@ -0,0 +1,296 @@
+# Set Up Service Registration Protocol (SRP) Server-Client Connectivity With OT CLI
+
+OpenThread offers SRP server and client commands for use with a Thread network
+to perform such functions as registering and leasing services, and managing
+server and client hosts.
+
+This guide provides basic tasks that use some of the more common `srp server`
+and `srp client` commands.
+
+## SRP server commands
+
+For a list of `srp server` commands, type `help`:
+
+> srp server help
+addrmode
+auto
+disable
+domain
+enable
+help
+host
+lease
+seqnum
+service
+state
+Done
+
+## SRP client commands
+
+For a list of `srp client` commands, type `help`:
+
+> srp client help
+autostart
+callback
+help
+host
+keyleaseinterval
+leaseinterval
+service
+start
+state
+stop
+ttl
+Done
+
+## CLI Command Reference
+
+For descriptions and syntax of all commands, refer to the CLI Command Reference.
+The SRP server commands begin alphabetically with
+[srp server (enable,disable)](https://openthread.io/reference/cli/commands#srp_server_enabledisable).
+The SRP client commands begin alphabetically with
+[srp client autostart (get)](https://openthread.io/reference/cli/commands#srp_client_autostart_get).
+
+## Thread Border Router codelab
+
+The [OpenThread Border Router codelab](https://openthread.io/codelabs/openthread-border-router#1)
+includes information about how to set up the SRP client end device.
+
+## Overviews of some basic SRP commands
+
+The following commands are a subset of the `srp server` and `srp client` that
+can be used in sequence (example in the next section) to perform typical SRP tasks:
+
+1. `srp server enable` enables the SRP server once you have created the Thread network.
+
+1. `srp client host name` sets the client host name when the host is
+   not registered with the server. 
+
+1. `srp client host address (set)` either enables auto host client address mode or
+   explicitly sets the list of host client addresses.
+
+1. `srp client service add` adds  a service with a given instance name, service
+    name, and port number.
+
+1. `srp client autostart enable` enables auto-start mode. You can also manually
+   start the client by running `srp client start`. 
+
+1. `srp client host` and `srp client service` provide status about whether
+   the client host and service have been successfully registered on the client node.
+
+1. `srp client service remove` removes a service but retains the service name.
+
+1. `srp client host remove` removes the host, service, and their respective names.
```

Comment:
```suggestion
1. `srp client host remove` removes the host and all registered services.
```

And tehcnically the names may not removed (unregistered) with server (due to "key lease" mechanism).